src/main/resources/webinterface/css/style.css

```diff
@@ -36,6 +36,7 @@ body {
 }
 #map .Bomb.explored {
 	background-color: #000000 !important;
+    font-size: 1.5em !important;

```

Comment:
Can you confirm this is the case in your text editor? My text editor tells me everything is fine. I have also written part of the css using the same text editor before, so I would be highly confused if it suddenly decided to no longer understand tabs.